Task_objects/index.js

```diff
@@ -1,24 +1,61 @@
-const cat = {
-  name: "Bertie",
-  breed: "Cymric",
-  color: "white",
-  greeting: function () {
-    console.log("Meow!");
-  },
-};
-
-// Write the code for 1st task below
-
-
```

Comment:
The use of bracket notation to access the 'name' property is unnecessary. You can directly assign the value of 'cat.name' to 'catName' variable.